.github/workflows/all_solutions.yml

```diff
@@ -77,6 +77,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Setup .NET 9 Preview
+        uses: actions/setup-dotnet@6bd8b7f7774af54e05809fcc5431931b3eb1ddee # v4.0.1
+        with:
+          dotnet-version: '9.0.x'
+          dotnet-quality: 'preview'
```

Comment:
Is the ga version available for installation instead of the preview? And what are the odds that GH updates their runners before this PR completes?? 🤣

Reply:
I just created a ticket to go back and clean these things up in a few weeks.